Source/WebCore/css/CSSSelector.cpp

```diff
@@ -731,18 +756,22 @@ String CSSSelector::selectorText(StringView separator, StringView rightSide) con
         } else if (cs->match() == Match::PseudoElement) {
             switch (cs->pseudoElementType()) {
             case CSSSelector::PseudoElementSlotted:
-                builder.append("::slotted(");
+                builder.append("::slotted("_s);
```

Comment:
Curious what the practical difference of this is.

Reply:
It creates a named literal instead of a char[], it's better optimized in WebKit in general.